### DISCUSSION TITLE
[Sunsetting] Dappkit

### DISCUSSION THREAD

**Discussion by nicolasbrugneaux** [celo-org/celo-monorepo#10221] [Q&A] [answered]
Hello Celo devs :wave:

We are sunsetting Dappkit due being barely used and generally superseded by WalletConnect (which v2 just came out!)

After syncing with Valora, we are seeing next to no usage of Dappkit. If you are using Dappkit at the moment and don’t know how to migrate to WalletConnect, please contact us via [Github Discussions ](https://github.com/celo-org/celo-monorepo/discussions/10221).

If you have a particularly sensitive issue please reach out to [devrel@celo.org](mailto:devrel@celo.org).

The timeline is yet to be fully decided, but expect the dappkit project to be deleted from the monorepo in the coming weeks.

[Copied from our forum post [here](https://forum.celo.org/t/sunsetting-dappkit/5336)]

**Comment by thoms994** [comment]
I believe that the project did not work because too little content was regularly created for their Dapp and it’s capacity.

**Comment by nicolasbrugneaux** [comment, marked as answer]
The package was deleted from the mono-repo and deprecated on npm.
